src/header/header.scss

```diff
@@ -265,4 +292,13 @@
       }
     }
   }
+
+  .iui-button-split-menu {
+    height: 100%;
+    padding: 0 0 0 $iui-xs;
+
+    .iui-button {
+      padding: 0 $iui-xs;
+    }
+  }
```

Comment:
Much cleaner code right now 👍 Need to decide about active state button.

Reply:
I added active split button